dev-haskell/static-bytes/static-bytes-0.1.1.ebuild

```
@@ -0,0 +1,29 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.9.0.0
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="A Haskell library providing types representing 8, 16, 32, 64 or 128 bytes of data"
```

Shorten this to <=80 bytes

```suggestion
DESCRIPTION="A Haskell library providing types representing various byte sizes of data"
```